src/library.js

```diff
@@ -4192,7 +4192,7 @@ LibraryManager.library = {
     var trace = _emscripten_get_callstack_js();
     var parts = trace.split('\n');
     for (var i = 0; i < parts.length; i++) {
-      var ret = Runtime.dynCall('iii', [0, arg]);
+      var ret = Runtime.dynCall_iii(func, 0, arg);
```

Comment:
Yeah, this was added for Rust but we never found a good way to test it. I wouldn't worry about it.